enterprise/backend/src/metabase_enterprise/metabot_v3/tools/api.clj

```diff
@@ -388,7 +402,7 @@
   [:merge
    ::basic-metric
    [:map {:decode/tool-api-response #(update-keys % metabot-v3.u/safe->snake_case_en)}
-    [:queryable_dimensions ::columns]]])
+    [:queryable_dimensions {:optional true} ::columns]]])
```

Comment:
This change might require a change on the AI service. As long as `with_metric_queryable_dimensions` is not set to false, this property will be returned.